app/services/participants/action.rb

```diff
@@ -10,8 +10,8 @@ class Action
     attribute :course_identifier
 
     validates :lead_provider, presence: true
-    validates :participant, presence: true
-    validates :course_identifier, inclusion: { in: Course::IDENTIFIERS }, allow_blank: false
+    validates :participant, presence: { message: I18n.t(:invalid_participant) }
```

Comment:
Can we keep these in `errors/models`? I think there are a few other places that hard code them in the models but the majority are inferred and it would be good to get them all consistent:

https://github.com/DFE-Digital/npq-registration/blob/main/config/locales/en.yml#L61